hack/pp-cm-helper.sh

```diff
@@ -0,0 +1,192 @@
+    else # self managed azure
+        [[ ! ${AZURE_SUBNET_ID} ]] && [[ ${AZURE_SUBSCRIPTION_ID} ]] && [[ ${AZURE_RESOURCE_GROUP} ]] && \
+            AZURE_SUBNET_ID="/subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}/providers/Microsoft.Network/virtualNetworks/${AZURE_RESOURCE_GROUP%-rg}-vnet/subnets/${AZURE_RESOURCE_GROUP%-rg}-worker-subnet"
+        [[ ! ${AZURE_NSG_ID} ]] && [[ ${AZURE_SUBSCRIPTION_ID} ]] && [[ ${AZURE_RESOURCE_GROUP} ]] && \
+            AZURE_NSG_ID="/subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}/providers/Microsoft.Network/networkSecurityGroups/${AZURE_RESOURCE_GROUP%-rg}-nsg"
+    fi
+    AZURE_INSTANCE_SIZE=${AZURE_INSTANCE_SIZE:-Standard_B2als_v2}
+    AZURE_INSTANCE_SIZES=${AZURE_INSTANCE_SIZES:-Standard_B2als_v2,Standard_D2as_v5,Standard_D4as_v5,Standard_D2ads_v5}
+    #AZURE_IMAGE_ID=${AZURE_IMAGE_ID}
+}
+
+function userVerification() {
+    echo && echo "###### Setting Values (press Enter for the [suggested] value, \"drop\" to remove key)"
+    verifyAndSetVars "${common_vars[@]}"
+    case $cld in
+        "aws")
+            verifyAndSetVars "${aws_vars[@]}"
+            verifyAndSetVars "${aws_optional[@]}"
+            ;;
+        "azure")
+            verifyAndSetVars "${azure_vars[@]}"
+            verifyAndSetVars "${azure_optional[@]}"
+            ;;
+        *)
+            error_exit "Invalid provider";;
+    esac
+    verifyAndSetVars "${custom_vars[@]}"
+}
+
+function applyCM() {
+    echo && echo "###### Applying"
+    [[ -n $YES ]] || (read -r -p "Apply the Changes to the peer-pods-cm ConfigMap? [y/N] " && [[ "$REPLY" =~ ^[Yy]$ ]]) || exit 0
+    ${CLI} delete cm peer-pods-cm -n openshift-sandboxed-containers-operator > /dev/null 2>&1
+    ${CLI} create cm peer-pods-cm --from-env-file=${envFile} -n openshift-sandboxed-containers-operator
+    until ${CLI} get cm peer-pods-cm -n openshift-sandboxed-containers-operator >/dev/null 2>&1 >/dev/null ; do
+       echo "Waiting for ConfigMap to be created..."
+       sleep 1
+    done
+    echo && echo "###### Done"
+    ${CLI} get cm peer-pods-cm -n openshift-sandboxed-containers-operator -o jsonpath='{.data}' | jq
+}
+
+function initialization() {
+    CLI=$(command -v oc) || CLI=$(command -v kubectl) || error_exit "Missing k8s client"
+
+    command -v jq > /dev/null || error_exit "jq is required"
+
+    ${CLI} cluster-info &> /dev/null || error_exit "No reachable cluster"
+
+    ${CLI} get ns openshift-sandboxed-containers-operator &> /dev/null || error_exit "Namespace doesn't exist yet, install OSC first"
+
+    # TODO: allow also k8s clusters
+    cld=$(${CLI} get infrastructure -n cluster -o jsonpath='{.items[*].status.platformStatus.type}' | awk '{print tolower($0)}' | tr -d '"' ) && cld=${cld//none/libvirt}
+    oc get clusters.aro.openshift.io cluster &> /dev/null && aro=yes # mark as aro
```

Comment:
as convention can we use caps for global aro (eg. IS_ARO)?
You can set the default to false and set it true here